### ISSUE TITLE
Holster sign

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10223]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
 PMID: 31279808 : Holster sign Symmetric poikiloderma of hips and lateral thighs below the greater trochanter. May be reticulated, livedoid, or linear and is reported to be highly specific for DM
 PMID 32644343: Holster sign: poikiloderma involving the lateral aspects of the thighs

**Parent term (use hpo.jax.org/app)**
Poikiloderma HP:0001029

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:221 Dermatomyositis

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000006
- label: Holster sign
- parents: HP:0001029
- pmid: PMID:31279808|PMID:32644343
- def: Symmetric poikiloderma of hips and lateral thighs below the greater trochanter.
- comment: The Holster sign may indicate dermatomyositis.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10223
- synonyms:
